runtime/caml/mlvalues.h

```diff
@@ -471,6 +493,36 @@ CAMLextern value caml_set_oo_id(value obj);
 #define Caml_out_of_heap_header(wosize, tag)                           \
         Caml_out_of_heap_header_with_reserved(wosize, tag, 0)
 
+
+/* Deprecated/discouraged suppport for unsafe encoded exceptions.
```

Comment:
```suggestion
/* Obsolete -- support for unsafe "encoded" exceptions.
```
Deprecated is ambiguous as it has a specific meaning of "candidate for removal in a future version of the language" in some standardized languages. Since encoded exceptions were a public documented API and correct programs using it can be expected to exist, I think you mean obsolete/discouraged.